README.md

```diff
@@ -64,7 +64,7 @@ Also, check the [example](example) directory.
 | include | `string`/`array` | required | One or more paths that Sentry CLI should scan recursively for sources. It will upload all `.map` files and match associated `.js` files. |
 | org | `string` | optional | The slug of the Sentry organization associated with the app. |
 | project | `string` | optional | The slug of the Sentry project associated with the app. |
-| authToken | `string` | optional | The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/settings/account/api/auth-tokens/. |
+| authToken | `string` | optional | The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/settings/account/api/auth-tokens/. Required scopes: `project:releases` and `org:read` (only if `setCommits` option is used). |
```

Comment:
```suggestion
| authToken | `string` | optional | The authentication token to use for all communication with Sentry. Can be obtained from https://sentry.io/settings/account/api/auth-tokens/. Required scopes: `project:releases` (and `org:read` if `setCommits` option is used). |
```

Just makes it clearer that the note about `setCommits` only applies to `org:read`.